docs/bin/testing_formatter.sh

```diff
@@ -20,7 +20,12 @@ $(for test in $(../../bin/ansible-test sanity --list-tests --allow-disabled); do
 
 EOF
 
+SHA_CMD="sha1sum"
```

Comment:
Are you able to put a comment here to say why this is done so people looking at this in the future can understand why

Reply:
Quick comment added, please let me know if more detail should be added.